gradle.properties

```
@@ -0,0 +1,12 @@
+org.gradle.caching=true
+group=io.ballerina.stdlib
+version=3.0.0-SNAPSHOT
+
+checkstylePluginVersion=10.12.0
+spotbugsPluginVersion=5.0.14
+shadowJarPluginVersion=8.1.1
+downloadPluginVersion=5.4.0
+releasePluginVersion=2.8.0
+testngVersion=7.6.1
+eclipseLsp4jVersion=0.12.0
+ballerinaGradlePluginVersion=2.1.8
```

```suggestion
ballerinaGradlePluginVersion=2.2.0
```
with improvement to the Ballerina Gradle plugin